lilypad/server/models/prompts.py

```diff
@@ -46,19 +46,21 @@ class GeminiCallArgsCreate(BaseModel):
     stop_sequences: list[str] | None = None
 
 
-class OpenAICallArgsCreate(BaseModel):
-    """OpenAI call args model."""
+class OpenAICallParams(BaseModel):
+    """OpenAI call args model.
+    https://platform.openai.com/docs/api-reference/chat/create
```

Comment:
nit: google docstring style needs an additional newline here between the short and long descriptions